pygeo/parameterization/DVGeoMulti.py

```diff
@@ -1508,8 +1508,8 @@ def update(self, ptSetName, delta):
             # Compute denominators for the integral evaluations
             # Add an epsilon so that these terms never become zero
```

Comment:
Looks good, but maybe this comment should also be updated slightly now that we use min?